README.md

```diff
@@ -21,18 +29,106 @@ $ kubectl create secret docker-registry iterativeai \
     --docker-password=<password>
 ```
 
-Create a file `override.yaml` with the following content or edit `values.yaml`
+### Prepare a TLS secret
 
-```yaml
-imagePullSecrets:
-  - name: iterativeai
+It is desired to enable Studio access via the `https` protocol
+(as opposed to `http`). This requires setting up
+a TLS secret for access (whether self-signed or "real").
```

Comment:
Let's mention that if the user already has a TLS certificate,  they should jump straight to the section about creating the secret